packages/grafana-api-clients/src/generator/plopfile.ts

```diff
@@ -107,15 +107,20 @@ export default function plopGenerator(plop: NodePlopAPI) {
     return actions;
   };
 
-  // Read files from data/openapi directory and use as an array of API clients that the user can select
-
-  const openapiDir = path.join(basePath, 'data/openapi');
-  let possibleOpenAPISpecs: string[] = [];
-  try {
-    possibleOpenAPISpecs = fs.readdirSync(openapiDir).filter((file: string) => file.endsWith('.json'));
-  } catch (e) {
-    possibleOpenAPISpecs = [];
-  }
+  const getOpenAPISpecs = (isEnterprise: boolean): string[] => {
+    const openapiDir = isEnterprise
+      ? path.join(basePath, 'pkg/extensions/apiserver/tests/openapi_snapshots')
+      : path.join(basePath, 'pkg/tests/apis/openapi_snapshots');
+
+    try {
+      const files = fs.readdirSync(openapiDir).filter((file: string) => file.endsWith('.json'));
+      return files;
+    } catch (e) {
+      throw new Error(
+        "No OpenAPI specs found! Are you trying to generate an API client for enterprise but haven't linked your local environment?"
+      );
```

Comment:
@Clarity-89 I tweaked it slightly so we actually throw an error if we couldn't load the specs. I imagine that if we did fail to load the snapshots, the likely scenario is that the user hadn't set up their enterprise env properly?

Reply:
Yeah, it makes sense 👍